tests/func/test_repro.py

```diff
@@ -829,19 +829,19 @@ def test(self):
         self.dvc.run(
             fname=stage,
             outs=[fname],
-            cmd=f"echo $SHELL > {fname}",
+            cmd=f"echo $HOME > {fname}",
```

Comment:
For the record: discussed that this change pretty much breaks the test, there should be another way to make it work in GHA.